src/features/output/OutputView.vue

```diff
@@ -39,27 +173,111 @@ export default {
         }
       ).addTo(this.map);
     },
-    // This function needs the input of the geoTiffUrl variable to take a tf and visualize it on the map
+    checkLayerGetsFoundWithMessage: function (layer) {
+      if (layer == null) {
```

Comment:
Ich würde an dieser Stelle den strict equality operator (`===`) verwenden:
```js
if (layer === null) {
// oder einfach nur
if (!layer) {
```
Was sog. strict operator sind und warum man diese nutzen sollte, kannst du hier nachlesen:
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons